qgeotilefetchergooglemaps.cpp

```diff
@@ -49,6 +50,11 @@ QGeoTileFetcherGooglemaps::QGeoTileFetcherGooglemaps(const QVariantMap &paramete
         _language = (langs.length() > 0) ? langs[0] : "en-US";
     }
 
+	// High DPI map tiles
+	if (qApp) {
+		m_scale = (qApp->devicePixelRatio() > 1) ? 2 : 1;
+	}

```

Comment:
@mfbernardes 
how about turning this into a plugin parameter instead of making it automatic?
`googlemaps.maps.scale`

see how i did it here:
https://github.com/vladest/googlemaps/pull/12

using `QApplication` to detect high DPI display would work, but i don't think we should force it the plugin source code. see [1].

Reply:
yes, agree. it would be better to do not add extra modules and pass this as a parameter

Reply:
@vladest thanks for confirming!
@mfbernardes let me know if you need help with turning this into a parameter.

Reply:
@neolit123 you’re definitely right. I’ll turn it into a parameter as suggested. I’ll post an updated patch until tomorrow.